com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/Launcher.java

```diff
@@ -178,7 +178,7 @@ void blockingDrain() throws IOException {
     return result;
   }
 
-  protected Drainer captureStdErr(Process p) {
+  protected Drainer captureStdErr(Process p) throws IOException {
```

Comment:
why this change?

Reply:
This change was related to the first solution but should be removed as it is irrelevant now.